### PR TITLE
Add attachments endpoint to mobile SM API

### PR DESCRIPTION
**Pull request by patrickvinograd**
## Description of change
Adds the attachments controller/serializer allowing secure message attachments to be downloaded, and to be properly rendered as sub-resources in a message. Swagger docs added for the newly-available endpoint.

## Things to know about this PR

Tested locally.

### CODE REVIEW

File: modules/mobile/docs/openapi.yaml [hunk 1]
```diff
@@ -1985,6 +1985,40 @@ paths:
             application/json:
               schema:
                 $ref: "./schemas/Errors.yml"
+  /v0/messaging/health/messages{id}/attachments/{attachment_id}:
```
Comment (kreek) [inline modules/mobile/docs/openapi.yaml:1988]:
```suggestion
  /v0/messaging/health/messages/{id}/attachments/{attachment_id}:
```

Comment (kreek) [reply modules/mobile/docs/openapi.yaml:1988]: added this suggestion but you'll need to render again 😄

**Review by kreek — APPROVED** [review]
LGTM 🚀

File: modules/mobile/docs/openapi.yaml (hunk 1 above)
Comment (patrickvinograd) [reply modules/mobile/docs/openapi.yaml:1988]: Good catch, thank you!

**Pull request merged**